weltyt.py

```diff
@@ -1,6 +1,115 @@
 import sqlite3
 
+def initializationDB(cursor):
 
+        with open("TYTDB.sql") as base:
+                for line in base:
+                        cursor.execute(line)
+
+lands_offert = {"humus": {"class": "humus", "id": 1, "growth_rate": 5, "price_ISL": 500, "plants": 0}, 
+                "humus_sag": {"class": "humus_sag", "id": 2, "growth_rate": 4, "price_ISL": 400, "plants": 0}, 
+                "brown_soil": {"class": "brown_soil", "id": 3, "growth_rate": 3, "price_ISL": 350, "plants": 0}, 
+                "lessive_soil": {"class": "lessive_soil", "id": 4, "growth_rate": 2, "price_ISL": 250, "plants": 0},
+                "clay_gravel": {"class": "clay_gravel", "id": 5, "growth_rate": 1, "price_ISL": 200, "plants": 0}}
+#INSERT INTO lands VALUES ("humus", 1, 5, 500, 0)
 print("Welcome to tyton")
 conn = sqlite3.connect("TYTDB.db")
 
+cur = conn.cursor()
+
+initializationDB(cur)        
+
+conn.commit()
+
+for row in cur.execute('SELECT * FROM resources'):
+        print(row)
+
+for rows in cur.execute('SELECT * FROM lands'):
+        print(rows)
+
+
+buying_plants = input("To grow tobbaco you have to buy some plants. If you want do it, write \"Plants\" please. ")
+if buying_plants == "Plants":
+    amount = int(input("How many plants do you want to buy? Write only integer please. Overwise weltyt stops working. "))
+
+    if type(amount) == int:
+        money_amount = cur.execute('SELECT amount  FROM resources WHERE type = "Money" ').fetchone()[0]
+        plants_price = cur.execute('SELECT price FROM resources WHERE type = "Plants" ').fetchone()[0]
+        new_money_amount =  money_amount - amount * plants_price
+        cur.execute(f'UPDATE resources SET amount = {new_money_amount}  WHERE type = "Money"; ') 
+
+        plants_amount = cur.execute('SELECT amount  FROM resources WHERE type = "Plants" ').fetchone()[0]
+        new_plants_amount = plants_amount + amount 
+        cur.execute(f'UPDATE resources SET amount = {new_plants_amount}  WHERE type = "Plants"; ')     
+    else:
+        print("Invalid input")
```

Comment:
You can just leave a nice message that the user is stupid and should provide a number.